db/sg_replicate_cfg.go

```diff
@@ -148,6 +153,7 @@ type ReplicationUpsertConfig struct {
 	Cancel                 *bool       `json:"cancel,omitempty"`
 	Adhoc                  *bool       `json:"adhoc,omitempty"`
 	BatchSize              *int        `json:"batch_size,omitempty"`
+	RunAs                  *string     `json:"run_as,omitempty"`
```

Comment:
I think `RunAs` needs to be included in the `ActiveReplicatorConfig.CheckpointHash()` method - as it affects what data is replicated - and we cannot rely on the correctness of checkpoints if that value changes.